NebulaHost/PlayerManager.cs

```diff
@@ -70,6 +70,28 @@ public void SendPacketToAllPlayers<T>(T packet) where T : class, new()
             }
         }
 
+        public void SendPacketToLocalPlanet<T>(T packet) where T : class, new()
+        {
+            foreach (Player player in GetConnectedPlayers())
+            {
+                if (player.Data?.LocalPlanetId == GameMain.localPlanet?.id)
```

Comment:
Maybe I'm wrong, but I think that right now we might not update the `LocalPlanetId` in the `PlayerData` after the initial game load.

Reply:
yeah, I think it is 0 for the new `PlayerData` and it gets updated at the first `PlayerMovement` packet. For the players, that are reconnecting, it should be already populated from the previous saved state.

I guess I can set the `LocalPlanetId` to the spawn planet for the new players.